src/layout/SimpleTable/SimpleTableComponent.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+import { Link, Table } from '@digdir/designsystemet-react';
+import dot from 'dot-object';
+
+import { Caption } from 'src/components/form/Caption';
+import { useExternalApi } from 'src/features/externalApi/useExternalApi';
+import { ErrorList } from 'src/layout/GenericComponent';
+import { useNodeItem } from 'src/utils/layout/useNodeItem';
+import type { PropsFromGenericComponent } from 'src/layout';
+import type { ColumnConfig } from 'src/layout/SimpleTable/config.generated';
+
+export const SimpleTableComponent = ({ node }: PropsFromGenericComponent<'SimpleTable'>) => {
+  const nodeItem = useNodeItem(node);
+
+  const { data: externalApi } = useExternalApi(nodeItem.data.id);
+  const data: unknown = dot.pick(nodeItem.data.path, externalApi);
+
+  if (!isArrayOfObjects(data)) {
+    return (
+      <ErrorList
+        nodeId={node.id}
+        errors={['Tabelldata må være en liste av objekter']}
+      />
+    );
```

Comment:
Instead of rendering out an ErrorList, just throw an error instead. It will be caught in the error handler, logged to devtools, and then displayed in an error list. I would prefer not to export the ErrorList, as it just makes it possible to circumvent the normal way these errors are caught and logged.